extensions/wikia/AdEngine/js/video/player/porvata/porvataPlayerFactory.js

```diff
@@ -40,6 +41,10 @@ define('ext.wikia.adEngine.video.player.porvata.porvataPlayerFactory', ['wikia.l
 					width = newWidth;
 					height = newHeight;
 				}
+				if (!width || !height) {
+					width = params.container.offsetWidth;
+					height = params.container.offsetHeight;
+				}
```

Comment:
Why do we want to set height & width based on offset when one of (newWidth, newHeight) will be 0?